### DISCUSSION TITLE
integrate data from different conditions

### DISCUSSION THREAD

**Discussion by AnneDi** [satijalab/seurat#4315] [Q&A] [answered]
Hello Seurat Team,

I have a problem to find the best solution about integrating/analysing my scRNA seq data.
Some facts that you should know:
I've performed 2 experiments. 1) control condition and 2  time points of viral infection condition. By analysing the first experiment (RunTsne) it revealed nice clustering and especially at later time point differences became clear within the sample conditions (Interferon signalling etc.).
So we decided to create knockout for ifnar1 to block/reduce this classy interferone signalling.
2) control conditions (each for wildtype and knockout cells) plus viral infection conditions (each in wildtype and knockout cells). (later time point to exp 1)

Now, after integrating all this different samples (SCTransform, RunUmap...) data do not show this obvious variation.

My question is now if this is the proper way to analyse these data. Or do you think I'd rather should split my data into 2 datasets (eg. comparison of wildtype cells with control and infection condition and 2nd comparison of latest timepoint where I can directly compare wildtype/knockout vs infected/noninfected with each other)?
I don't know what is common. I would appreciate a lot if you could point me in the right direction.

Thanks a lot in advance!

Best,
Anne

**Comment by timoast** [comment, marked as answer]
I would suggest taking a look at the [introductory integration vignette](https://satijalab.org/seurat/articles/integration_introduction.html), which demonstrates the integration and analysis of two treatment conditions, which is conceptually similar to your experiment.

**Reply by lkqnaruto** [reply]
I wonder if integration would be the right tool if I have 20 samples, 4 normal, 8 treatment1, 8 treatment2. And my goal is to find out cell types and DEG?
